tests/countries/test_trinidad_and_tobago.py

```diff
@@ -0,0 +1,243 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.constants import OPTIONAL
+from holidays.countries.trinidad_and_tobago import TrinidadAndTobago, TT, TTO
+from tests.common import CommonCountryTests
+
+
+class TestTrinidadAndTobago(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        years = range(1963, 2050)
+        super().setUpClass(TrinidadAndTobago, years=years)
```

Comment:
To account for our new Observed ones.
```suggestion
        super().setUpClass(TrinidadAndTobago, years=years, years_non_observed=years)
```